.github/workflows/autoconf-check.yml

```diff
@@ -12,7 +12,8 @@ jobs:
   debian-family:
     strategy:
       matrix:
-        version: [sid, testing, stable]
+        version: [testing, stable]
+        #version: [sid, testing, stable]
```

Comment:
If the messages have changed, then I think the tests should be written more relaxed to accomodate that instead of ignoring this. Tests that overly rely on specifics of the environment are brittle, and might cause more harm than do good.

Reply:
This was to test if that was the case, the annoying thing with GHA is that is stops testing if its aware that another test in the same file has failed. Because `sid` is the first one it stops the other ones.

So my plan was to properly fix this, but since you last reviewed I found out that the tests were never started so they work locally but not in GHA.